cmd/cmd.go

```diff
@@ -33,9 +33,15 @@ func Execute() {
 	}
 
 	root := NewQriCommand(EnvPathFactory, os.Stdin, os.Stdout, os.Stderr)
+	// If the subcommand hits an error, don't show usage or the error, since we'll show
+	// the error message below, on our own. Usage is still shown if the subcommand
+	// is missing command-line arguments.
+	root.SilenceUsage = true
+	root.SilenceErrors = true
+	// Execute the subcommand
 	if err := root.Execute(); err != nil {
 		printErr(os.Stdin, err)
-		os.Exit(-1)
+		os.Exit(1)
```

Comment:
Note: The intention here is that, if a subcommand returns an error, bail out with os.Exit, but I don't believe this code was *ever* being run. Each subcommand was itself calling ExitIfErr, which calls os.Exit and kills the process. This entire change, which makes commands actually return an error, brings this code back into line with the apparent intention.